src/porepy/utils/comp_geom.py

```diff
@@ -1741,6 +1747,48 @@ def distance_segment_segment(s1_start, s1_end, s2_start, s2_end):
     dist = d_starts + sc * d1 - tc * d2
     return np.sqrt(dist.dot(dist))
 
+#------------------------------------------------------------------------------#
+
+def distance_point_segment(pt, start, end):
```

Comment:
I have implemented the same function, although allowing for multiple points and segments, see branch frac_mesh_3d. I suggest leaving this for now, but replacing it with the more general function when the latter branch is merged.